cli/wallet/wallet.go

```diff
@@ -20,13 +20,13 @@ import (
 )
 
 var (
-	errNoPath         = errors.New("target path where the wallet should be stored is mandatory and should be passed using (--path, -p) flags")
+	errNoPath         = errors.New("target path where the wallet should be stored is mandatory and should be passed using (--wallet, -w) flags")
 	errPhraseMismatch = errors.New("the entered pass-phrases do not match. Maybe you have misspelled them")
 )
 
 var (
 	walletPathFlag = cli.StringFlag{
-		Name:  "path, p",
+		Name:  "wallet, w",
```

Comment:
So where is the documentation update for this?

Reply:
You mean `docs/cli.md`? Fixed. I've first grepped by full name.

Reply:
Yep, that was missing.